src/styles/_catalog.scss

```diff
@@ -1,7 +1,7 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+  gap: 48px;
```

Comment:
Strange position, fix it 
![image](https://user-images.githubusercontent.com/57729155/134042820-38593514-204d-45b0-a272-3d4921238e3b.png)